CenterForOpenScience/osf.io
# Turn on institutional affiliation by default for registrations [ENG-2714]

## Purpose

Draft registrations should have the users institutional affiliation by default.
## Changes

Adding functionality and tests

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify all institutional affiliation is turned on by default for registrations
- Verify

What are the areas of risk? N/A

Any concerns/considerations/questions that development raised? N/A

## Documentation

N/A

## Side Effects

N/A
## Ticket

https://openscience.atlassian.net/browse/ENG-2714
